### PR TITLE
add cutlass@3.5.1

### PR DESCRIPTION
**Pull request by tongke6**
add cutlass@3.5.1 to bcr.

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (cutlass) have been updated in this PR. Please review the changes.

**Comment by tongke6** [comment]
@bazel-io skip_check unstable_url

File: modules/cutlass/3.5.1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 7.x
+  - 6.x
```
Comment (Vertexwahn) [inline modules/cutlass/3.5.1/presubmit.yml:10]: Add also `- rolling`

**Comment by tongke6** [comment]
Hello @Vertexwahn @meteorcloudy, could you please review the changes again? Thanks!

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**